### PR TITLE
Accreditation

### PR DESCRIPTION
**Pull request by mphprogrammer**

### CODE REVIEW

File: app/controllers/accreditations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    @accreditation.destroy
+
+    respond_to do |format|
+      format.html { redirect_to accreditations_url, 
+          notice: "#{@accreditation.institution} was successfully destroyed." }
+    end
+  end
+
+  #############################################################################
+  ## set_eight_key
```
Comment (plusjeff) [inline app/controllers/accreditations_controller.rb]: Typo?

File: app/controllers/accreditations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  ## Obtains the model instance from the id parameter.
+  #############################################################################  
+  def set_accreditation
+    @accreditation = Accreditation.find(params[:id])
+  end
+
+  #############################################################################
+  ## eight_key_params
```
Comment (plusjeff) [inline app/controllers/accreditations_controller.rb]: Typo?

File: app/controllers/accreditations_controller.rb (hunk 1 above)
Comment (mphprogrammer) [inline app/controllers/accreditations_controller.rb]: Missed updating the comment! - good catch.

File: app/controllers/accreditations_controller.rb (hunk 2 above)
Comment (mphprogrammer) [inline app/controllers/accreditations_controller.rb]: Missed updating the comment! - good catch.

File: app/views/accreditations/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<%= render partial: 'introduction', 
+		locals: { 
+			new_link: false, edit_link: false, 
+			list_link: false, destroy_link: false
+		} 
+%>
+
+<h2><small>Create New Scorecard Data</small></h2>
```
Comment (plusjeff) [inline app/views/accreditations/new.html.erb]: Typo? (`Scorecard`)

Comment (mphprogrammer) [inline app/views/accreditations/new.html.erb]: Missed changing the static content - thanks!

File: app/views/accreditations/_introduction.html.erb [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+<div class="row row-space-6">
+  <div class="col-xs-8">
+    From this data, the GI Bill Comparison Tool's database is built.
+  </div>
+
+  <div class="col-xs-4">
+    <% if list_link %>
+      <div class="row row-space-6">
+        <div class="col-xs-12">
+          <%= link_to 'List Scorecard Data',
```
Comment (plusjeff) [inline app/views/accreditations/_introduction.html.erb]: Scorecard typo

File: app/views/accreditations/_introduction.html.erb [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            class: "btn btn-primary btn-xs", 
+            role: "button" %>
+        </div>
+      </div>
+    <% end %>
+
+    <% if new_link %>
+      <div class="row row-space-6">
+        <div class="col-xs-12">
+          <%= link_to 'New Scorecard',
```
Comment (plusjeff) [inline app/views/accreditations/_introduction.html.erb]: Scorecard typo

Comment (mphprogrammer) [inline app/views/accreditations/_introduction.html.erb]: Got it!

**Comment by jeffmaher** [comment]
Tests out and LGTM after typos. Can merge after typo fixes.

**Pull request merged**